crates/optimism/rpc/Cargo.toml

```diff
@@ -12,13 +12,29 @@ workspace = true
 
 [dependencies]
 # reth
+revm.workspace = true
+revm-inspectors = { workspace = true, features = ["js-tracer"] }
+revm-primitives = { workspace = true, features = ["dev"] }
+reth-errors.workspace = true
+reth-evm.workspace = true
```

Comment:
we likely don't need all of those.

we can now enable this in this pr:

https://github.com/paradigmxyz/reth/blob/6e1521456623aa3dbec586e3aa48b642c1fe28d5/crates/optimism/rpc/src/lib.rs#L8-L8

Reply:
updated